File: baekjoon/1934/1934.py
'''
최소공배수 python 풀이
https://doctcoder.tistory.com/entry/%EC%B5%9C%EC%86%8C%EA%B3%B5%EB%B0%B0%EC%88%98-%ED%92%80%EC%9D%B4
'''
import math

n=int(input())
for i in range(n):
	a,b=map(int,input().split())
	gcds=math.gcd(a,b)
	print(int(a*b/gcds))

File: baekjoon/1934/1934.cpp
/*
A+B C++ 풀이
https://doctcoder.tistory.com/entry/AB-%ED%92%80%EC%9D%B4
*/
#include <iostream>
#include <numeric>

using namespace std;

int main()
{
	int t;
	cin>>t;
	for(int i=0;i<t;i++)
	{
		int a,b;
		cin>>a>>b;
		cout<<lcm(a,b)<<"\n";
	}
}
